L1Trigger/L1THGCal/plugins/ntuples/HGCalTriggerNtupleHGCDigis.cc

```diff
@@ -17,6 +17,7 @@ class HGCalTriggerNtupleHGCDigis : public HGCalTriggerNtupleBase
         ~HGCalTriggerNtupleHGCDigis(){};
         virtual void initialize(TTree&, const edm::ParameterSet&, edm::ConsumesCollector&&) override final;
         virtual void fill(const edm::Event& e, const edm::EventSetup& es) override final;
+        virtual void simhits(const edm::Event& e);
```

Comment:
No need to be a virtual function.
Also, better to be private than public (not used outside the class).